app/scripts/views/confirm_reset_password.js

```diff
@@ -139,21 +141,20 @@ define(function (require, exports, module) {
     },
 
     _finishPasswordResetSameBrowser (sessionInfo) {
-      // Only the account UID, unwrapBKey and keyFetchToken are passed
-      // from the verification tab. Load other from localStorage
-      var account = this.user.getAccountByUid(sessionInfo.uid);
-
+      const account = this.user.getAccountByUid(sessionInfo.uid);
+
+      // A bug in e10s causes localStorage in about:accounts and content tabs to be isolated from
+      // each other. Writes to localStorage from /complete_reset_password are not able to be read
+      // from within about:accounts. Because of this, all account data needed to sign in must
+      // be passed between windows. See https://github.com/mozilla/fxa-content-server/issues/4763
+      // and https://bugzilla.mozilla.org/show_bug.cgi?id=666724
+      //
       // keyFetchToken and unwrapBKey are sent from the verification tab,
```

Comment:
Is the part about Hello still needed here? Or maybe we can nuke this comment block completely?